src/SimpleStorage.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.17;
+
+import { PluginUUPSUpgradeable, IDAO } from "@aragon/osx/core/plugin/PluginUUPSUpgradeable.sol";
+
+/**
+ * @title Budget
+ * @author DAOBox (Security@DAOBox.app)
+ * @notice Budgeting module for efficient spending from an Aragon OSx DAO using allowance chains
+ * to delegate spending authority
+ */
+contract SimpleStorage is PluginUUPSUpgradeable {
+    bytes32 public constant STORE_PERMISSION_ID = keccak256("STORE_PERMISSION");
+
+    uint256 public number; // added in build 1
+
+    /// @notice Initializes the plugin when build 1 is installed.
+    /// @param _number The number to be stored.
+    function initialize(IDAO _dao, uint256 _number) external initializer {
+        __PluginUUPSUpgradeable_init(_dao);
+        number = _number;
+    }
+
+    /// @notice Stores a new number to storage. Caller needs STORE_PERMISSION.
+    /// @param _number The number to be stored.
+    function storeNumber(uint256 _number) external auth(STORE_PERMISSION_ID) {
+        number = _number;
+    }
+
+    /// @notice This empty reserved space is put in place to allow future versions to add new variables without shifting
+    /// down storage in the inheritance chain (see [OpenZepplins guide about storage
+    /// gaps](https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps)).
+    uint256[50] private __gap;
```

Comment:
```suggestion
    uint256[49] private __gap;
```

Reply:
This contract introduces `uint256 public number;`
Accordingly, you must decrease the gap by 1 so that this contract ends with a round number of storage slots.
50 slots per contract is the OZ standard.